impl/src/main/java/org/jboss/weld/injection/producer/NoopLifecycleCallbackInvoker.java

```diff
@@ -53,8 +53,4 @@ public boolean hasPreDestroyMethods() {
         return true; // we cannot say no for sure
     }
 
-    @Override
-    public boolean hasPostConstructMethods() {
```

Comment:
Don't remove this method but add `hasPostConstructCallback()` with `return false`.